src/components/Header/Header.tsx

```diff
@@ -29,8 +27,8 @@ export const Header: FC<HeaderProps> = () => {
           </div>
           {user ? (
             <div className={style.header_avatar_wrapper}>
-              <Image
-                src={img} // Заміна на аватар користувача
+              <img
+                src={user.photoURL || "./public/images/avatar"} // Заміна на аватар користувача
```

Comment:
Wrong path for image should be relative to domain you have related to this specific file ./ is current folder of this component file, you need different path here